### PR TITLE
modify Linux download info

### PR DESCRIPTION
**Pull request by cobalt2727**
Feedback welcome.

I've seen one too many people join the unofficial Dolphin Discord trying to figure out why the PPA doesn't work on their 20.04 installation

I'm not really sure what the system here is to handle other languages, honestly... Is that automatic?

### CODE REVIEW

File: dolweb/downloads/templates/downloads-index.html [hunk 1]
```diff
@@ -112,8 +112,20 @@ <h1>{% trans "Stable versions" %}</h2>
 
 <div id="other-linux">
     <h1>{% trans "Linux distributions" %}</h1>
-    <p>{% blocktrans %}Ubuntu users can install a PPA
-    for development and stable versions of Dolphin here: <a href="https://wiki.dolphin-emu.org/index.php?title=Installing_Dolphin#Ubuntu">Installing Dolphin</a>
+    <p>{% blocktrans %}Ubuntu users on 16.04 and 18.04 can install a PPA
+    for development and stable versions of Dolphin here - users on 20.04 or newer (or Linux Mint, Pop OS, Zorin, etc) must compile or use the Flatpak): <a href="https://wiki.dolphin-emu.org/index.php?title=Installing_Dolphin#Ubuntu">Installing Dolphin</a>
```
Comment (leoetlino) [inline dolweb/downloads/templates/downloads-index.html]:
```suggestion
    for development and stable versions of Dolphin here - users on 20.04 or newer (or Linux Mint, Pop OS, Zorin, etc.) must build Dolphin from source or use the Flatpak): <a href="https://wiki.dolphin-emu.org/index.php?title=Installing_Dolphin#Ubuntu">Installing Dolphin</a>
```

File: dolweb/downloads/templates/downloads-index.html [hunk 2]
```diff
@@ -112,8 +112,20 @@ <h1>{% trans "Stable versions" %}</h2>
...
+    {% endblocktrans %}</p>
+    <p>{% blocktrans %}
+    Arch Linux users can download Dolphin via <a href="https://archlinux.org/packages/community/x86_64/dolphin-emu/">pacman</a>
```
Comment (leoetlino) [inline dolweb/downloads/templates/downloads-index.html]: "Arch Linux, Fedora, and openSUSE can install the `dolphin-emu` package with their package manager. However, distro packages might be slightly out of date. Compile from source for the latest version."

so we don't need to repeat the same paragraph 3 times

**Comment by cobalt2727** [comment]
Hopefully I got the formatting right, I'm not too familiar with html and I don't have an environment set up to test right now

**Comment by cobalt2727** [comment]
I might have messed this up while rebasing. One sec...

**Comment by OatmealDome** [comment]
Merged #154 instead. This mentions the Flatpak version, and since we have no control over it, we probably shouldn't make it seem like we're endorsing them.

**Pull request closed without merging**